### PR TITLE
Rasters in different projections supported by tileindex TILESRS

### PR DESCRIPTION
**Pull request by pedros007**
Inspired by https://lists.osgeo.org/pipermail/mapserver-users/2019-November/081494.html

### CODE REVIEW

File: en/mapfile/projection.txt [hunk 1]
```diff
@@ -135,9 +135,18 @@ shapefile layers or raster layers :
...
-  refers to a tile index (OGR layer or shapefile tileindex), the projection will
-  be retrieved according to the above describe rules.
+  the projection will be retrieved from the GDAL datasource.
+  
+- If the raster layer refers to a tile index (OGR layer or shapefile
+  tileindex), the projection will be retrieved according to the above
+  describe rules.
+
+- If the raster layer refers to a tile index with rasters in a variety
+  of projections, the :code:`TILESRS` keyword can choose the
```
Comment (rouault) [inline en/mapfile/projection.txt]:
```suggestion
  of projections, the :code:`TILESRS` keyword can choose the
```
```suggestion
  of projections, the :code:`TILESRS` keyword should be specified to choose the
```

File: en/mapfile/projection.txt [hunk 2]
```diff
@@ -135,9 +135,18 @@ shapefile layers or raster layers :
...
+  appropriate project for the corresponding raster. See
```
Comment (rouault) [inline en/mapfile/projection.txt]:
```suggestion
  appropriate projection for the corresponding raster. See
```

Comment (pedros007) [reply en/mapfile/projection.txt]: Good catches, @rouault! This is what happens when I'm in a hurry to finish work before the weekend.

**Comment by pedros007** [comment]
Good catches, @rouault! This is what happens when I'm in a hurry to finish work before the weekend.

**Comment by jmckenna** [comment]
thanks @pedros007 !

**Comment by sdlime** [comment]
All, does this need to be applied to master as well?

**Comment by rouault** [comment]
> All, does this need to be applied to master as well?

Just did
```
git checkout master
git pull
git merge origin/branch-7-4
[solve conflict, git add modified file]
git commit -a
git push
```

**Pull request merged**